example/src/main/java/com/graphhopper/example/IsochroneExample.java

```diff
@@ -23,7 +24,7 @@ public static void main(String[] args) {
 
         // snap some GPS coordinates to the routing graph and build a query graph
         FastestWeighting weighting = new FastestWeighting(encoder);
-        Snap snap = hopper.getLocationIndex().findClosest(42.508679, 1.532078, new FiniteWeightFilter(weighting));
+        Snap snap = hopper.getLocationIndex().findClosest(42.508679, 1.532078, new FiniteWeightFilter(weighting, encodingManager.getBooleanEncodedValue(InSubnetwork.key("car"))));
```

Comment:
This case is a bit tricky. On the one hand it should (and is) entirely up to the user what `EdgeFilter` is used for the snapping, but on the other hand in this example its nice to suggest that subnetworks normally should be excluded. But in any case I don't think one should be forced to use InSubnetwork, just because the FiniteWeightFilter constructor requires it.

Reply:
And if we use a separate SnapEdgeFilter for all methods in LocationIndex?

Reply:
Can you have a look if the introduced DefaultSnapFilter (or DefaultSnappingFilter?) is something that works for you too?

Reply:
Yes I like it with the DefaultSnapFilter (no real preference regarding the name btw)